bucket/rw/console.py

```diff
@@ -84,19 +84,23 @@ def write(self, reading: Reading):
                     goal_table.add_row(goal.name, goal.description, str(goal.target))
 
             if self.write_points:
-                point_table = Table(*(axis_titles + point_table_base_columns), title=f"{point.name} - {point.description}")
+                standard_columns = [c.copy() for c in point_table_base_columns]
+                point_table = Table(*(axis_titles + standard_columns), title=f"{point.name} - {point.description}")
                 point_tables.append(point_table)
 
                 point_bucket_goals = reading.iter_bucket_goals(point.bucket_start, point.bucket_end)
                 point_bucket_hits = reading.iter_bucket_hits(point.bucket_start, point.bucket_end)
                 for bucket_goal, bucket_hit in zip(point_bucket_goals, point_bucket_hits):
                     goal = goals[bucket_goal.goal]
-                    hits = bucket_hit.hits
+                    
                     target = goal.target
 
                     if target > 0:
+                        hits = bucket_hit.hits
                         target_percent = f"{(min(target,hits) / target) * 100:.2f}%"
                     else:
+                        assert hits == 0, f"Expect hits to be zero, not {hits}"
```

Comment:
I know I suggested this assertion... but I think log an error instead

it would be really frustrating (even just for debugging) if you couldn't print out the coverage because of this